CHANGES.md

```diff
@@ -1,6 +1,12 @@
 ringpop-go changes
 ==================
 
+v0.x.y (to be released)
```

Comment:
Love the addition of this to the change log. Was thinking of exactly this so we keep track during development on what gets added. However I was thinking about calling this `Dev` since we don't know the version to be released. What are you thoughts on calling this section something like `Dev` or `Development`?